static/css/cdr_common.css

```diff
@@ -195,14 +199,24 @@ li {
 }
 
 /** Header menu */
+
 #mainmenu {
-	margin: 0 0 0 0px;
+	margin: 0;
+    background-color: #E6E9EC;
+    padding: 0;
 }
 
-#mainmenu > li {
+#mainmenu > li
+/*, .submenu > li */
+{
 	float: left;
-	margin: 10px 0 0 0;
-	text-indent: 0px;
+	margin: 0;
+	text-indent: 0;
+    background-color: #E6E9EC;
+	border-right: 2px solid #053a53;
```

Comment:
I would suggest doing this with margin-right rather than a colored border since that causes dark lines to appear between the tabs in the admin ui